kv/task.go

```diff
@@ -362,35 +362,31 @@ func (s *Service) findTasksByOrg(ctx context.Context, tx Tx, filter influxdb.Tas
 		return nil, 0, influxdb.ErrUnexpectedTaskBucketErr(err)
 	}
 
-	c, err := indexBucket.Cursor()
+	prefix, err := org.ID.Encode()
 	if err != nil {
-		return nil, 0, influxdb.ErrUnexpectedTaskBucketErr(err)
+		return nil, 0, influxdb.ErrInvalidTaskID
 	}
 
-	var k, v []byte
+	key := prefix
 	// we can filter by orgID
 	if filter.After != nil {
-		key, err := taskOrgKey(org.ID, *filter.After)
+		key, err = taskOrgKey(org.ID, *filter.After)
 		if err != nil {
 			return nil, 0, err
 		}
-		// ignore the key:val returned in this seek because we are starting "after"
-		// this key
-		c.Seek(key)
-		k, v = c.Next()
-	} else {
-		// if we dont have an after we just move the cursor to the first instance of the orgID
-		key, err := org.ID.Encode()
-		if err != nil {
-			return nil, 0, influxdb.ErrInvalidTaskID
-		}
 
-		k, v = c.Seek(key)
+		// append a extra character because we want to skip "after" record
+		key = []byte(string(key) + "\x00")
```

Comment:
This is making me think we need a handy option like `kv.SkipFirstKey()` or `kv.WithoutFirstKey()`

Reply:
that would be a nice addition :)